docs/aws/ContinuousDeployment.rst

```diff
@@ -13,16 +13,20 @@ You can enable continuous deployment of the changes in your source repository to
 
    It is optional to keep the deployment in your account automatically synchronized with the source code repository.
 
-For the continuous deployment to work, you need to fork the source code to register the webhook listener that triggers an upgrade of your deployment.
+For the continuous deployment to work, complete the following steps:
 
-After forking, make sure to update the ``repository.url`` in the :file:`package.json` file in your fork.
+1. Fork the source code to register the webhook listener that triggers an upgrade of your deployment.
 
-This sets up an AWS CodePipeline, which triggers a CodeBuild project for every push to the :file:`saga` branch.
-You can configure the branch in the ``deploy.branch`` property of the :file:`package.json` file.
-The CodeBuild project upgrades the CloudFormation stack, which contains the nRF Asset Tracker resources.
+#. Make sure to update the ``repository.url`` in the :file:`package.json` file in your fork.
+
+   This sets up an AWS CodePipeline, which triggers a CodeBuild project for every push to the ``saga`` branch.
+   
+#. Configure the branch in the ``deploy.branch`` property of the :file:`package.json` file.
+
+   The CodeBuild project upgrades the CloudFormation stack, which contains the nRF Asset Tracker resources.
```

Comment:
```suggestion
#. Configure the branch in the ``deploy.branch`` property of the :file:`package.json` file.
   The CodeBuild project upgrades the CloudFormation stack, which contains the nRF Asset Tracker resources.
```

Reply:
I prefer the output with the empty line that separates the output comment from the actual step. So, keeping this as it is.